EmulatorPkg/EmulatorPkg.dsc

```diff
@@ -537,15 +537,15 @@
   MSFT:NOOPT_*_*_CC_FLAGS = /Od /Oy-
   GCC:DEBUG_CLANGPDB_*_CC_FLAGS =-O0 -Wno-unused-command-line-argument -Wno-incompatible-pointer-types -Wno-enum-conversion -Wno-incompatible-pointer-types -Wno-sometimes-uninitialized -Wno-constant-conversion -Wno-main-return-type
 
-  MSFT:*_*_*_DLINK_FLAGS     = /ALIGN:4096 /FILEALIGN:4096 /SUBSYSTEM:CONSOLE
-  MSFT:DEBUG_*_*_DLINK_FLAGS = /EXPORT:InitializeDriver=$(IMAGE_ENTRY_POINT) /BASE:0x10000
-  MSFT:NOOPT_*_*_DLINK_FLAGS = /EXPORT:InitializeDriver=$(IMAGE_ENTRY_POINT) /BASE:0x10000
+  MSFT:*_*_*_DLINK_FLAGS     = /ALIGN:4096 /FILEALIGN:4096
+  MSFT:DEBUG_*_*_DLINK_FLAGS = /BASE:0x10000
+  MSFT:NOOPT_*_*_DLINK_FLAGS = /BASE:0x10000
 
 !if $(WIN_HOST_BUILD) == TRUE
   #
   # CLANGPDB tool chain depends on WIN_HOST_BUILD flag to generate the windows application.
   #
-  GCC:*_CLANGPDB_*_DLINK_FLAGS     = /ALIGN:4096 /FILEALIGN:4096 /SUBSYSTEM:CONSOLE
-  GCC:DEBUG_CLANGPDB_*_DLINK_FLAGS = /EXPORT:InitializeDriver=$(IMAGE_ENTRY_POINT) /BASE:0x10000
-  GCC:NOOPT_CLANGPDB_*_DLINK_FLAGS = /EXPORT:InitializeDriver=$(IMAGE_ENTRY_POINT) /BASE:0x10000
+  GCC:*_CLANGPDB_*_DLINK_FLAGS     = /ALIGN:4096 /FILEALIGN:4096
+  GCC:DEBUG_CLANGPDB_*_DLINK_FLAGS = /BASE:0x10000

```

Comment:
Why was the EXPORT removed?

Reply:
@mdkinney , the `EXPORT` was only for WinHost.inf, but specifying it here made it true for all binaries, which produced a linker warning that this was not the `__stdcall` interface that the linker expected for an export, which for the FW modules itself it is not, it is `__cdecl`. So, this is removed from applying to all modules built in EmulatorPkg to only WinHost.inf, which is the only valid module to apply this. This removes the linker warning (and continues to boot EmulatorPkg).

Reply:
This is incorrect.  At one time, source level debug from VS required the export.  Has that been resolved in a different way?

Reply:
Let me follow up on this. I did not try source level debug from VS after this, just booting EmulatorPkg.

Reply:
@mdkinney , paged back the information on this. This InitializeDriver based source level debugging was removed a while ago and VS source level debugging has been broken on EmulatorPkg (regardless of my change above) unless you cherry-pick this change: https://github.com/tianocore/edk2/pull/4874. I'm not sure why this change never went in, I did follow up with @nate-desimone , but I did not hear a response.

With Nate's change and my change, I can do source level debugging on VS EmulatorPkg.

Reply:
I can do source level debug of EmulatorPkg with VS Code without any changes from current HEAD

Reply:
Interesting, I took my statement from Nate's PR, but I also can do source level debugging from top of tree.